src/Integration/EnvironmentIntegration.php

```diff
@@ -28,7 +28,10 @@ public function setupOnce(): void
 
             if (null !== $integration) {
                 $event->setRuntimeContext($integration->updateRuntimeContext($event->getRuntimeContext()));
-                $event->setOsContext($integration->updateServerOsContext($event->getOsContext()));
+
+                if (function_exists('php_uname')) {
```

Comment:
What about moving this condition inside the `updateServerOsContext` function? In fact, we could simply have an early return before [line 56](https://github.com/getsentry/sentry-php/blob/master/src/Integration/EnvironmentIntegration.php#L53)